core/tests/protractor_desktop/topicsAndSkillsDashboard.js

```diff
@@ -103,6 +103,7 @@ describe('Topics and skills dashboard functionality', function() {
       topicsAndSkillsDashboardPage.get();
       topicsAndSkillsDashboardPage.navigateToUnusedSkillsTab();
       topicsAndSkillsDashboardPage.mergeSkillWithIndexToSkillWithIndex(0, 0);
+      topicsAndSkillsDashboardPage.get();
```

Comment:
Why was this added? I'm wondering how this test ever passed if we needed this line

Reply:
Hi @U8NWXD, I think this has been added based on my comment [here](https://docs.google.com/document/d/1qEBLDr5vGM7LR5zdCBVTy2pLe60-WPaFWU7t6LNX5os/edit). I'm fine with any of the alternative approaches I've mentioned there as well. PTAL!

Regarding why it didn't happen before, it seems like after the migration, there is a small delay before the topics tab is selected right after merging a skill. I think it is probably because we are manually triggering $rootScope.$apply() in the migrated service (this is the current pattern we follow see 6(c) and 7(c) [here](https://github.com/oppia/oppia/wiki/Angular-Migration#implementation-details-to-migrate-services)). 

The reasoning behind me being OK with get() is that, I don't see why the tabs must switch in the first place. If there are multiple skills in the `unused skills tab`, it makes sense to stay in the same tab in case the user wants to merge more skills to existing ones. @aks681, @seanlip what are your thoughts on this (ie. Letting the "Unused skills" tab stay selected after merging a skill)?

Reply:
I'm fine with letting the tab stay selected.

Reply:
Cool, thanks! So in that case, we could have the test manually navigate to topics tab to check if the skill has successfully merged as part of this test. @U8NWXD does that sound reasonable (we could replace get() with a "topicsTab.click()" or similar since that is clearer)? I'll create an issue to fix the navigation thing.

Reply:
That sounds good to me!